src/frontend/index.tsx

```diff
@@ -1,25 +1,38 @@
 import 'iframe-resizer/js/iframeResizer.contentWindow';
 
 import { Grommet } from 'grommet';
+import jwt_decode from 'jwt-decode';
```

Comment:
`jwtDecode`? As it's a default export we can name it however we please and we don't use snake case